docs/source/chapt_vector_support/example1/Python_Model_Vectors_Handling.rst

```diff
@@ -0,0 +1,180 @@
+This would correspond to the number of elements in the vector, if size is
+greater than 1.
+
+.. figure:: ../figs/add_ip_vector_var_2.png
+   :alt: Figure 3: Specify size of the input vector variable
+   :name: fig.add.ip.vector.var.2
+
+   Figure 3: Specify size of the input vector variable
+
+*Step 4*: In the next user interface prompt, enter 0 as the minimum value for
+the variable elements, shown in Figure 4, and click ok.
+
+.. figure:: ../figs/add_ip_vector_var_3.png
+   :alt: Figure 4: Specify minimum value of the input vector variable elements
+   :name: fig.add.ip.vector.var.3
+
+   Figure 4: Specify minimum value of the input vector variable elements
+
+*Step 5*: In the next user interface prompt, enter 1 as the maximum value for
+the variable elements, shown in Figure 5, and click ok.
+
+.. figure:: ../figs/add_ip_vector_var_4.png
+   :alt: Figure 5: Specify maximum value of the input vector variable elements
+   :name: fig.add.ip.vector.var.4
+
+   Figure 5: Specify maximum value of the input vector variable elements
+
+*Step 6*: In the next user interface prompt, enter 0.5 as the value for the
+variable elements, shown in Figure 6, and click ok.
+These are the values at which the simulation would run.
+
+.. figure:: ../figs/add_ip_vector_var_5.png
+   :alt: Figure 6: Specify value of the input vector variable elements
+   :name: fig.add.ip.vector.var.5
+
+   Figure 6: Specify value of the input vector variable elements
+
+At this point in the implementation, the 'Input Variables' section of the node
+editor is populated with the vector variable elements x_0 through x_5,
+(which correspond to vector variable 'x') as shown in Figure 7.
+
+.. figure:: ../figs/add_ip_vector_var_6.png
+   :alt: Figure 7: Input vector variable elements in Node Panel
+   :name: fig.add.ip.vector.var.6
+
+   Figure 7: Input vector variable elements in Node Panel
+
+*Step 7*: Under 'Output Variables' section, click on '+' to add a new output
+variable.
+Further, in the user interface prompt, enter the variable name 'y' as shown in
+Figure 8, and click ok.
+
+.. figure:: ../figs/add_op_var.png
+   :alt: Figure 8: Add output scalar variable 'y'
+   :name: fig.add.op.var
+
+   Figure 8: Add output scalar variable 'y'
+
+*Step 8*: In the next user interface prompt, enter 1 as the size of the variable
+(since it is a scalar), as shown in Figure 9, and click ok.
+
+.. figure:: ../figs/add_op_var_2.png
+   :alt: Figure 9: Specify size of the output variable
+   :name: fig.add.op.var.2
+
+   Figure 9: Specify size of the output variable
+
+At this point in the implementation, the 'Output Variables' section of the node
+editor is populated with the scalar variable y as shown in Figure 10.
+
+.. figure:: ../figs/add_op_var_3.png
+   :alt: Figure 10: Output variable in Node Panel
+   :name: fig.add.op.var.3
+
+   Figure 10: Output variable in Node Panel
+
+*Step 9*: Enter the rosenbrock function in the node script (set to 'post' mode)
```

Comment:
Rosenbrock should be capitalized here.

Reply:
Thanks @eslickj I will make this change.